adapter/internal/oasparser/model/policy_container.go

```diff
@@ -41,6 +41,18 @@ const (
 	policyValTypeMap    string = "Map"
 )
 
+var (
+	// policyDefFuncMap is a map of functions used in policy definitions
+	policyDefFuncMap = template.FuncMap{
+		// isParamExists checks the key is exists in the params map, this will not consider the value of the param
+		// if the go templated "{{ if .param }}" is used, that will consider the value of the param (if value is a zero value, it consider as not exists)
```

Comment:
We can break the comment into two lines

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/2754/commits/b7f6b18870e58303c07c2b9ff89afef8a3735571